binscripts/rvm-installer

```diff
@@ -783,27 +783,33 @@ follow this link for details how to fix:
 
 rvm_install_validate_volume_mount_mode()
 {
-  \typeset path partition
+  \typeset path partition test_exec
 
   path=$rvm_path
 
-  # $rvm_path might not exists at this point so we need to traverse the directory tree upwards
-  # to find the name of the partition and then check if it has not been mount with noexec mode
+  # Directory $rvm_path might not exists at this point so we need to traverse the tree upwards
   while [[ -n "$path" ]]
```

Comment:
I was also thinking about `mkdir -p` but this would check if we can create the path also - which would require more error handling and could leave garbage directories if user decides not to use this path... I'm fine with traversing the path down, in the end write permissions are required to create the directory so the script should be able to also create the script, it's easier to cleanup one file.